chain/chain.go

```diff
@@ -44,26 +44,34 @@ type RPCClient struct {
 // but must be done using the Start method.  If the remote server does not
 // operate on the same decred network as described by the passed chain
 // parameters, the connection will be disconnected.
-func NewRPCClient(chainParams *chaincfg.Params, connect, user, pass string, certs []byte,
-	disableTLS bool) (*RPCClient, error) {
+func NewRPCClient(chainParams *chaincfg.Params, connect, user, pass string,
```

Comment:
This function signature may not be modified in this module's major version.  You will have to find a different, backwards-compatible way of adding this, or make a v2 module.

Reply:
fixed